ios/Podfile.lock

```diff
@@ -472,7 +472,7 @@ PODS:
     - React-Core
   - react-native-image-manipulator (1.0.5):
     - React
-  - react-native-image-picker (4.8.5):
+  - react-native-image-picker (4.10.1):
```

Comment:
Pretty sure this change should have been committed in another PR that updated this dependency.